### DISCUSSION TITLE
Tabs don't seem to respect theme.normal.tab_color and theme.active.tab_color

### DISCUSSION THREAD

**Discussion by vendion** [herbstluftwm/herbstluftwm#1520] [Q&A] [answered]
Using herbstluftwm 0.9.5 with the max layout tabs, the tabs doesn't seem to respect the `theme.normal.tab_color` and `theme.active.tab_color` attributes.  As an example, I have `theme.normal.tab_color` set to #4c566a and `theme.active.tab_color` set to #5e81ac (and `theme.urgent.tab_color` to #bf616a). Unfortunately, it appears that these values are getting ignored and only the active color is used.

![2022-09-08-19:05:01-screenshot](https://user-images.githubusercontent.com/145018/189236180-23e35ecf-7e59-4e51-9e58-3502714e9ec9.png)

Note the split in the image is via Kitty, but even if I don't use Kitty terms splits I get the same result.

**Comment by t-wissmann** [comment, marked as answer]
`tab_color` always refers to "the color of non-urgent and unfocused tabs" (from the docs). This means that

  - `theme.active` specifies all colors needed to render the decoration of focused clients, hence:
  - `theme.active.tab_color` is the color of unfocused tabs in the focused client's decoration.
  - `theme.active.color` is the color of the focused tab in the focused client's decoration
  - `theme.normal.color` is the color of the focused tab in unfocused clients' decorations
  - `theme.normal.tab_color` is the color of unfocused tabs in unfocused clients' decorations

Does this help?

**Reply by vendion** [reply]
Okay, so `theme.{active|normal}.color` and `theme.{active|normal}.tab_color` should be different values.
